app/assets/config/manifest.js

```diff
@@ -1,3 +1,2 @@
 //= link_tree ../images
-//= link_directory ../javascripts .js
-//= link_directory ../stylesheets .css
+//= link application.css
```

Comment:
https://github.com/rails/sprockets/blob/main/UPGRADING.md#manifestjs

Looks like from this documentation, the `javascript` part _is_ ok to remove.